docker-compose.yml

```diff
@@ -3,18 +3,40 @@
 version: '3'
 
 services:
+
+  main-db:
+    image: postgres:14-alpine
+      # Docs for options to the postgres server command:
+    # https://www.postgresql.org/docs/current/app-postgres.html
+    command: postgres -c "log_lock_waits=on" -N 1000 -c "fsync=off"
+    env_file: ./app/local.env
+    ports:
+      - "5432:5432"
+    platform: linux/amd64 # Fix running on Apple silicon
```

Comment:
```suggestion
    # Docs for options to the postgres server command: 
    # https://www.postgresql.org/docs/current/app-postgres.html 
    command: postgres -c "log_lock_waits=on" -N 1000 -c "fsync=off" 
    # Load environment variables for local development. 
    env_file: ./app/local.env 
    # Fix running on Apple silicon. 
    platform: linux/amd64
```

Reply:
I've updated everything as suggested with one minor caveat that I had to leave the port numbers for the DB so it's accessible outside the container.